src/components/modals/FlipInputModal.tsx

```diff
@@ -251,9 +249,11 @@ export class FlipInputModalComponent extends React.PureComponent<Props, State> {
   }
 }
 
+const deviceHeight = Dimensions.get('window').height
+
 const getStyles = cacheStyles((theme: Theme) => ({
   hackContainer: {
-    flex: deviceHeight <= MINIMUM_DEVICE_HEIGHT ? 1 : 0
+    flex: deviceHeight <= 580 ? 1 : 0
```

Comment:
Why was the constant removed? I like that it described the value. Can we at least move it to the top of this file?